mlir/lib/Dialect/MQTOpt/Transforms/FromQuantumComputationPattern.cpp

```diff
@@ -214,6 +247,15 @@ struct FromQuantumComputationPattern final : mlir::OpRewritePattern<AllocOp> {
       }
     }
 
+    // Now insert all the qubits back into the registers they were extracted
+    // from.
+    for (size_t i = 0; i < numQubits; i++) {
+      auto insertOp = createRegisterInsert(
+          newAlloc, currentRegister, currentQubitVariables[i], i, rewriter);
+      // Keep track of qubit register access
+      currentRegister = insertOp.getOutQureg();
```

Comment:
```suggestion
      currentRegister = insertOp.getOutQreg();
```
Since #933 will now merge before this PR, I think this line has to change after the merge.